packages/cli-e2e/utils/cli.ts

```diff
@@ -61,7 +61,7 @@ export async function setupUIProject(
   cliProcesses: ChildProcessWithoutNullStreams[],
   options: ISetupUIProjectOptionsArgs = {}
 ) {
-  const uiProjectFolderName = 'ui-projects';
+  const uiProjectFolderName = '../ui-projects';
```

Comment:
We need to move the ui projects outside the e2e-cli directory or we get warnings about different versions of Jest.